tests/queries/0_stateless/02020_exponential_smoothing.reference

```diff
@@ -654,23 +654,23 @@ exponentialTimeDecayedAvg
 0	48	0.201	████████████████████
 0	49	0.196	███████████████████▌
 Check `exponentialTimeDecayed.*` supports sliding windows
-2	1	3.010050167084	2	3.030251507111	0.993333444442
-1	2	7.060905027605	4.080805360107	4.02030134086	1.756312382816
-0	3	12.091654548833	5.101006700134	5.000500014167	2.418089094006
-4	4	11.050650848754	5.050250835421	5.000500014167	2.209909172572
-5	5	9.970249502081	5	5.000500014167	1.993850509716
-1	6	20.07305726224	10.202013400268	5.000500014167	4.014210020072
-0	7	15.991544871125	10.100501670842	3.98029867414	4.017674596889
+2	1	2.950447180363	1.960397346614	2.970248507056	0.993333444442
```

Comment:
Why result of `exponentialTimeDecayedAvg` is not changed? I assumed that changing window would affect result of all function. Is avg special in this context? How?

Reply:
It was not changed because current row is the same thing as last value in window for this query.